sdk/programmableconnectivity/Azure.Communication.ProgrammableConnectivity/samples/LocationSample.md

```diff
@@ -0,0 +1,19 @@
+# Location verify
+
+```C#
+using Azure.Communication.ProgrammableConnectivity;
+
+private readonly Uri _endpoint = new Uri("https://eastus.prod.apcgatewayapi.azure.com");
+private const string ApcGatewayId = "/subscriptions/abcdefgh/resourceGroups/dev-testing-eastus/providers/Microsoft.programmableconnectivity/gateways/apcg-eastus";
+
+DeviceLocation client = ProgrammableConnectivityClient(_endpoint, DefaultAzureCredential()).GetDeviceLocationClient()
+DeviceLocationVerificationContent deviceLocationVerificationContent = new DeviceLocationVerificationContent(new NetworkIdentifier("NetworkCode", "Telefonica_Brazil"), 80.0, 85.1, 50, new LocationDevice
+{
+    PhoneNumber = "+5551980449999",
```

Comment:
Since I'm using the sample from the test, I'd rather keep this as is, and assume that the reader will know to put their number in.

Can I keep as is, or should I be using `if SNIPPET` etc in the test code to get the desired behaviour?

Reply:
make it `500000000`